### ISSUE TITLE
train RNet report error : empty tensor

### ISSUE THREAD

**Issue by emily2he** [AITTSMD/MTCNN-Tensorflow#25]
Hi  @AITTSMD
     Thank you for your great work!
      I want to train MTCNN according to your introduction. repeat for times. but when I train RNet model, I met a issue，error shows an empty tensor size is zero, so I print the data, showing bbox_batch_array is all zero, I don't know the reason. 
      could you help me giving me some advice? thank you!

error as below:
-------------------------------------------------------------------------------------------------------------
image_batch_array (384, 24, 24, 3)
label_batch_array (384,)
bbox_batch_array (384, 4)
landmark_batch_array (384, 10)
label_batch_array -2.0
bbox_batch_array [[ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]
 [ 0.  0.  0.  0.]]
landmark_batch_array [ 0.22222221  0.27160493  0.75925922  0.33950618  0.58641976  0.67901236
  0.17283952  0.86419755  0.67901236  0.91358024]
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Reshape cannot infer the missing input size for an empty tensor unless all specified input sizes are non-zero
	 [[Node: gradients/TopKV2_grad/Reshape = Reshape[T=DT_INT32, Tshape=DT_INT32, _device="/job:localhost/replica:0/task:0/gpu:0"](TopKV2/_351, gradients/TopKV2_grad/stack)]]
----------------------------------------------------------------------------------------------------------------

**Comment by AITTSMD** [comment]
@emily2he hi,please pay attention that if label == -2,then the data is used for landmark detection and   we should not use its bounding box's information.

**Comment by emily2he** [comment]
@AITTSMD thank you for your reply. 
I understand the label =-2 is for landmark. I saw the code for training PNet and RNet is similar, PNet can be trained successfully with same train function and similar files of neg,pos,part and landmark.but it is failed when trained RNet, If that how to modify for that?

**Comment by emily2he** [comment]
@AITTSMD I print the log of training PNet. when label ==-2, the box's information is still zero, but doesn't report error, it is ok when training PNet, so what's the reason cause the different I don't understand. if convenient , we can communicate with qq(mine:275490132). thank you very much.
----------------------------------------------------------------------------------------------
image_batch_array (384, 12, 12, 3)
label_batch_array (384,)
bbox_batch_array (384, 4)
landmark_batch_array (384, 10)
label_batch_array -2.0
bbox_batch_array [[ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.18000001 -0.07       -0.02        0.13      ]
 [ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.          0.          0.          0.        ]
 [ 0.08        0.01       -0.09        0.31999999]]
landmark_batch_array [ 0.34523809  0.25        0.72619045  0.20238096  0.70238096  0.47619048
  0.35714287  0.76190478  0.71428573  0.73809522]
----------------------------------------------------------------------------

**Comment by emily2he** [comment]
@AITTSMD thank you for your reply.
I have solved this issue, the mainly problem is that the data for rnet is not right. some item and filename need be modified. thank you!
